<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Log_screen</class>
 <widget class="QMainWindow" name="Log_screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>370</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_login{
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(20,47,78,219), stop:1 rgba(85,98,112,226));
	color:rgba(255,255,255,210);
	border-radius: 5px;
}
QPushButton#button_login:hover{
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(40,67,98,219), stop:1 rgba(105,118,132,226));
}
QPushButton#button_login:pressed{
	padding-left:5px;
	padding-stop:5px;
	background-color: rgba(105,118,132,200);
}
QPushButton#button_cancer{
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(20,47,78,219), stop:1 rgba(85,98,112,226));
	color:rgba(255,255,255,210);
	border-radius: 5px;
}
QPushButton#button_cancer:hover{
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(40,67,98,219), stop:1 rgba(105,118,132,226));
}
QPushButton#button_cancer:pressed{
	padding-left:5px;
	padding-stop:5px;
	background-color: rgba(105,118,132,200);
}
QPushButton#button_show:pressed{
	padding-left:5px;
	padding-stop:5px;
	background-color: rgba(105,118,132,200);
}
QPushButton#button_pass:pressed{
	padding-left:5px;
	padding-stop:5px;
	background-color: rgba(105,118,132,200);
}
QPushButton#button_error{
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(20,47,78,219), stop:1 rgba(85,98,112,226));
	color:rgba(255,255,255,210);
	border-radius: 5px;
}
QPushButton#button_error:hover{
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(40,67,98,219), stop:1 rgba(105,118,132,226));
}
QPushButton#button_erroe:pressed{
	padding-left:5px;
	padding-stop:5px;
	background-color: rgba(105,118,132,200);
}
</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>300</width>
       <height>420</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/background.png);
border-radius:20px;

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>280</width>
       <height>390</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,100);
border-radius:20px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>95</y>
       <width>101</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgba(255,255,255,210);
</string>
     </property>
     <property name="text">
      <string>Log in</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="screen_user">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>165</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border : none;
border-bottom: 2px solid rgba(10,118,132,255);
color: rgba(255,255,255,230);
padding-bottom : 7px;</string>
     </property>
     <property name="placeholderText">
      <string> User Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="screen_pass">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>230</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border : none;
border-bottom: 2px solid rgba(10,118,132,255);
color: rgba(255,255,255,230);
padding-bottom : 7px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string> Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_login">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>280</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>L O G  I N</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_cancer">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>330</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>C A N C E R</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>380</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poor Richard</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 85, 127);</string>
     </property>
     <property name="text">
      <string>MAGICIAN ROBOT</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>430</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgba(255,255,255,140);</string>
     </property>
     <property name="text">
      <string>Creted by Tháº¯ng</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_show">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>240</y>
       <width>51</width>
       <height>28</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255,255,0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>eye (1).png</normaloff>eye (1).png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="button_pass">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>240</y>
       <width>51</width>
       <height>28</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255,255,0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>eye-crossed.png</normaloff>eye-crossed.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="button_error">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>280</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>A G A I N</string>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>label_6</zorder>
    <zorder>screen_user</zorder>
    <zorder>screen_pass</zorder>
    <zorder>button_cancer</zorder>
    <zorder>label_7</zorder>
    <zorder>label_8</zorder>
    <zorder>button_show</zorder>
    <zorder>button_pass</zorder>
    <zorder>button_error</zorder>
    <zorder>button_login</zorder>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
